chia/wallet/nft_wallet/uncurry_nft.py

```diff
@@ -0,0 +1,98 @@
+from chia.types.blockchain_format.program import Program
+from chia.wallet.puzzles.load_clvm import load_clvm
+
+SINGLETON_TOP_LAYER_MOD = load_clvm("singleton_top_layer_v1_1.clvm")
+NFT_MOD = load_clvm("nft_innerpuz.clvm")
+
+
+class UncurriedNFT:
+    """
+    A simple solution for uncurry NFT puzzle.
+    Initial the class with a full NFT puzzle, it will do a deep uncurry.
+    This is the only place you need to change after modified the Chialisp curried parameters.
+    """
```

Comment:
https://peps.python.org/pep-0257/

I like semantic newlines.  I don't offhand know if there are any issues with them in docstrings or how others feel about them.

It looks like there are a few other places to match this format.  Thanks, by the way, for caring about docs.  `:]`

```suggestion
    """A simple solution for uncurry NFT puzzle.

    Initial the class with a full NFT puzzle, it will do a deep uncurry.
    This is the only place you need to change after modified the Chialisp curried parameters.
    """
```

Reply:
I think a new line is more friendly with doc generators and human reading